pdp_util/raster.py

```diff
@@ -68,8 +69,11 @@ def __call__(self, environ, start_response):
                     content_type='application/json',
                     charset='utf-8')
             return res(environ, start_response)
+
         else:
-            return super(RasterServer, self).__call__(environ, start_response)
+            orca_url = build_orca_url(self.config['handlers'], self.config['thredds_root'], req)
```

Comment:
If we don't end up using the `path_info` below, then we can just pass in `req.query_string`.